PYME/IO/tabular.py

```diff
@@ -546,6 +546,29 @@ def __init__(self, filename):
         self.res = scipy.io.loadmat(filename)  # TODO: evaluate why these are cast as floats
         
         self._keys = [k for k in self.res.keys() if not k.startswith('_')]
+
+        # Check for multicolor
```

Comment:
I'd be inclined to make the multicolour (cell array) stuff it's own data source.

Reply:
Although not as convinced on this one as some of the other comments